plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -122,6 +126,7 @@ class NonMetadataPackage(UnitMixin, FileContentUnit):
     release = mongoengine.StringField(required=True)
     checksum = mongoengine.StringField(required=True)
```

Comment:
Yes I can see that now. Can a story be put in redmine to remove these old things with `backwards compatible = True`. That way we can at least track the removal.